boards/apollo3/redboard_artemis_atp/Makefile

```diff
@@ -44,4 +44,5 @@ flash-app:
 
 .PHONY: test
 test:
-	$(Q)OBJCOPY=${OBJCOPY} PORT=$(PORT) $(CARGO) test $(NO_RUN) --bin $(PLATFORM) --release
+	$(Q)OBJCOPY=${OBJCOPY} PORT=$(PORT) $(CARGO) test $(NO_RUN) \
```

Comment:
Do we need these two makefile changes?

Reply:
It was part of a larger cleanup, but a similar change got merged separately. So this is now just what's left.

It does make it easier to read though, before it was a really long line